docs/csharp/language-reference/compiler-messages/cs0116.md

```diff
@@ -11,7 +11,7 @@ ms.assetid: 4cb137b5-ec29-4c1a-adde-9f8424cb9496
 
 A namespace does not directly contain members such as fields or methods  
   
- Inside a [namespace](../../../csharp/language-reference/keywords/namespace.md), the compiler only accepts classes, structures, unions, enumerations, interfaces, and delegates. This error is often generated by developers from a C/C++ background who forget that in C#, methods and variables must be declared and defined within a struct or class. For more information, see [General Structure of a C# Program](../../../csharp/programming-guide/inside-a-program/general-structure-of-a-csharp-program.md).  
+ Inside a [namespace](../keywords/namespace.md), the compiler only accepts another namespaces, classes, structures, enumerations, interfaces, and delegates. This error is often generated by developers from a C/C++ background who forget that in C#, methods and variables must be declared and defined within a struct or class. For more information, see [General Structure of a C# Program](../../../csharp/programming-guide/inside-a-program/general-structure-of-a-csharp-program.md).  
```

Comment:
> another namespaces

This phrasing isn't quite right (English) grammatically. "another" is expecting a singular noun but you've provided a plural one.

I don't have any sense of the C# grammar that is being discussed.

Reply:
@richlander thanks for the correction; I think "another" can be dropped at all.